Makefile

```diff
@@ -32,6 +31,8 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 
 test :
+    python ./setup.py develop
+    py.test
```

Comment:
This does not work on a fresh virtualenv. The setup.py develop won't install py.test.
A workaround is to run python setup.py test (which seems to works without requiring develop or any virtualenv)